src/Export/VTK.jl

```diff
@@ -57,13 +57,13 @@ Return a `DatasetFile` that data can be appended to, see
 The keyword arguments are forwarded to `WriteVTK.vtk_grid`, see 
 [Data Formatting Options](https://juliavtk.github.io/WriteVTK.jl/stable/grids/syntax/#Data-formatting-options)
 """
-function WriteVTK.vtk_grid(filename::AbstractString, grid::Union{Grid{dim,<:Any,T}, NonConformingGrid{dim,<:Any,T}}; kwargs...) where {dim,T}
+function WriteVTK.vtk_grid(filename::AbstractString, grid::AbstractGrid{dim}; kwargs...) where {dim}
     cls = MeshCell[]
     for cell in getcells(grid)
         celltype = Ferrite.cell_to_vtkcell(typeof(cell))
         push!(cls, MeshCell(celltype, nodes_to_vtkorder(cell)))
     end
-    coords = reshape(reinterpret(T, getnodes(grid)), (dim, getnnodes(grid)))
+    coords = reshape(reinterpret(eltype(getnodes(grid,1).x), getnodes(grid)), (dim, getnnodes(grid)))
```

Comment:
I declared the dispatch for now on `AbstractGrid` which should be fine. However, the `eltype(getnodes(grid,1).x)` should be done better

Reply:
`get_coordinate_eltype`?

Reply:
like this?
```suggestion
    coords = reshape(reinterpret(get_coordinate_eltype(getnodes(grid,1)), getnodes(grid)), (dim, getnnodes(grid)))
```

Reply:
ah sorry I was confused by the other `get_coordinate_type` function

Reply:
Yea, we need to fix the docstring.